app/assets/stylesheets/modules/admin.scss

```diff
@@ -0,0 +1,27 @@
+.admin-requests {
+  h3 {
+    color: $sul-admin-h3-color;
+    font-size: 28px;
+  }
+
+  h4 {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    font-size: 16px;
+    font-style: italic;
+  }
+
+  table {
+    margin: 10px 0 40px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `10px 0 40px` instead of `10px 0 40px 0`